app/models/report_generators/system_performance/fy2019/measure_six.rb

```diff
@@ -132,11 +130,13 @@ def measure_6_a_and_b
       @answers[:sixab_b6][:value] = project_exists_from[:ph].count
       @answers[:sixab_b7][:value] = @answers[:sixab_b2][:value] + @answers[:sixab_b3][:value] + @answers[:sixab_b4][:value] + @answers[:sixab_b5][:value] + @answers[:sixab_b6][:value]
 
-      @support[:sixab_b2][:support] = support_for(answer: :sixab_b2, data: project_exists_from[:so])
-      @support[:sixab_b3][:support] = support_for(answer: :sixab_b3, data: project_exists_from[:es])
-      @support[:sixab_b4][:support] = support_for(answer: :sixab_b4, data: project_exists_from[:th])
-      @support[:sixab_b5][:support] = support_for(answer: :sixab_b5, data: project_exists_from[:sh])
-      @support[:sixab_b6][:support] = support_for(answer: :sixab_b6, data: project_exists_from[:ph])
+      client_personal_ids = personal_ids(project_exit_scope.pluck(:client_id))
+
+      @support[:sixab_b2][:support] = support_for(answer: :sixab_b2, data: project_exists_from[:so], personal_ids: client_personal_ids)
+      @support[:sixab_b3][:support] = support_for(answer: :sixab_b3, data: project_exists_from[:es], personal_ids: client_personal_ids)
+      @support[:sixab_b4][:support] = support_for(answer: :sixab_b4, data: project_exists_from[:th], personal_ids: client_personal_ids)
+      @support[:sixab_b5][:support] = support_for(answer: :sixab_b5, data: project_exists_from[:sh], personal_ids: client_personal_ids)
+      @support[:sixab_b6][:support] = support_for(answer: :sixab_b6, data: project_exists_from[:ph], personal_ids: client_personal_ids)
```

Comment:
Shouldn't these all use different sets of personal_ids?  Maybe something needs to be pulled out of project_exists_from[:so], project_exists_from[:es], etc?

Reply:
The clients in `project_exists_from` are projected from `project_exit_scope`, so they should all be included in `client_personal_ids`.